controllers/controllers/resource/template.go

```diff
@@ -106,6 +107,17 @@ func (r *VsphereTemplate) TemplateResources(ctx context.Context, eksaCluster *an
 		}
 	}
 
+	// pin cgroupDriver to systemd for k8s >= 1.21
```

Comment:
can we add a comment specifying we won't need this when the controller follows cp first and md second upgrade order?